connectors/cassandra/cassandra-api/pom.xml

```diff
@@ -0,0 +1,33 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
```

Comment:
You added the api project but this is not added in the parent pom.xml file, thus the build is failing to find the `cassandra-api` module. Add this to https://github.com/teiid/teiid/blob/master/connectors/pom.xml#L91

Reply:
To package this into WildFly you need also reference here https://github.com/teiid/teiid/blob/master/wildfly/connectors/cassandra/feature-pack-cassandra/src/main/resources/modules/system/layers/dv/org/jboss/teiid/resource-adapter/cassandra/main/module.xml#L18  

This module stuff is little complicated, just add the API module there, that should be enough

Reply:
you mean adding "casandra-api" to the cassandra module's pom, right? I think you have linked the wrong pom here.

Reply:
yes